GitUI/UserControls/FileStatusList.cs

```diff
@@ -976,17 +999,29 @@ private void UpdateFileStatusListView(bool updateCausedByFilter = false)
                     FileStatusListView.Groups.Add(group);
                 }
 
-                foreach (var item in i.statuses)
+                if (hasChanges && i.statuses.Count < 1)
```

Comment:
Move the variable to the inner block and improve the readability of the condition:
```suggestion
                IReadOnlyList<GitItemStatus> itemStatuses;
                if (hasChanges && i.statuses.Count == 0)
```